backend/src/main/scala/cromwell/backend/BackendLifecycleActorFactory.scala

```diff
@@ -32,7 +50,7 @@ trait BackendLifecycleActorFactory {
                              ioActor: ActorRef,
                              backendSingletonActor: Option[ActorRef]): Props
 
-  def jobExecutionTokenType: JobExecutionTokenType = JobExecutionTokenType("Default", None)
+  def jobExecutionTokenType: JobExecutionTokenType = JobExecutionTokenType(name, None)
```

Comment:
Since we always want to override this, should we scrap this base implementation?